elastic/integrations
# ECS version update

- Enhancement


## What does this PR do?

Updates the ECS version to 8.5.1 from 1.12. The system and pipeline tests are executed to generate new sample events.

## Checklist

Refer #4792  for the checklist.

## How to test this PR locally

Run system and pipeline test for activemq and apache integrations and make sure nothing is breaking. Verify the changes with ecs release notes.

- Closes #4787 
- Closes #4785
